docs/sdk/serverless-signing/conditional-signing.md

```diff
@@ -5,123 +5,68 @@ import TabItem from '@theme/TabItem';
+Lit Actions inherit the powerful condition checking ability that Lit Protocol utilizes for [access control](../access-control/intro). You can easily check on or off-chain conditions inside of Lit Actions to generate proofs and condition-based transaction automations.
 
 The below example will check if the user has at least 1 Wei on Ethereum, only returning a signature if they do.
 
-:::note
-`toSign` data is required to be in 32 byte format. 
-
-The `ethers.utils.arrayify(ethers.utils.keccak256(...)` can be used to convert the `toSign` data to the correct format.
-:::
-
-Installed the latest client on `datil-dev`
+## Prerequisites
 
-```bash
-yarn install @lit-protocol/lit-node-client
-```
+- Knowlege of [SessionSigs](../authentication/session-sigs/intro)
+- Knowledge of how to [generate an AuthSig](../migrations/6.0.0.md#generate-an-authsig)
+- Basic understanding of [Lit Actions](../serverless-signing/quick-start)
 
-Set up the Lit Action code to be run on the Lit nodes.
+## Complete Code Example
 
-```jsx
-const litActionCode = `
-const go = async () => {
-  // test an access control condition
-  const testResult = await Lit.Actions.checkConditions({conditions, authSig, chain})
-
-  console.log('testResult', testResult)
-
-  // only sign if the access condition is true
-  if (!testResult){
-    return;
-  }
+The complete code example is available in the [Lit Developer Guides Code Repository](https://github.com/LIT-Protocol/developer-guides-code/tree/master/conditional-signing). There is both a browser and Node.js implementation of the code.
 
-  const message = new Uint8Array(
-    await crypto.subtle.digest('SHA-256', new TextEncoder().encode('Hello world'))
-  );
-  // this is the string "Hello World" for testing, hashed with sha-256 above.
-  const toSign = message;
-  // this requests a signature share from the Lit Node
-  // the signature share will be automatically returned in the HTTP response from the node
-  const sigShare = await LitActions.signEcdsa({ toSign, publicKey: "0x02e5896d70c1bc4b4844458748fe0f936c7919d7968341e391fb6d82c258192e64", sigName: "sig1" });
-};
+### Example Lit Action
 
+The below example will check if the user has at least 1 Wei on Ethereum, only returning a signature if they do. It uses the [`checkConditions`](https://actions-docs.litprotocol.com/#checkconditions) function from the [Lit Actions SDK](https://actions-docs.litprotocol.com/). This performs a conditional check on the user's Ethereum balance, checking the balance of the wallet address that signed the Sign-in With Ethereum [EIP-4361](https://eips.ethereum.org/EIPS/eip-4361) when creating the `AuthSig`, which is passed in as an argument to the Lit Action. The boolean returned will be used to determine whether the wallet will be used to sign `dataToSign` or not. 
```

Comment:
> This performs a conditional check on the user's Ethereum balance, checking the balance of the wallet address that signed the Sign-in With Ethereum [EIP-4361](https://eips.ethereum.org/EIPS/eip-4361) when creating the `AuthSig`, which is passed in as an argument to the Lit Action. The boolean returned will be used to determine whether the wallet will be used to sign `dataToSign` or not.

IMO this sentence is a bit complex. Consider "This function performs a conditional check on the authenticated user's Ethereum balance (the user who signed the Sign-in With Ethereum [EIP-4361](https://eips.ethereum.org/EIPS/eip-4361) message to create the `AuthSig`), returning a boolean. This is then used to determine whether or not the PKP will sign `dataToSign` or not."

Reply:
Yeah I like this better